comments.php

```diff
@@ -28,25 +28,13 @@
 		<h2 class="comments-title">
 			<?php
 				printf( // WPCS: XSS OK.
-					_nx( 'One thought on &ldquo;%2$s&rdquo;', '%1$s thoughts on &ldquo;%2$s&rdquo;', get_comments_number(), 'comments title', 'twentyseventeen' ),
+					_nx( 'One Reply to &ldquo;%2$s&rdquo;', '%1$s Replies to &ldquo;%2$s&rdquo;', get_comments_number(), 'comments title', 'twentyseventeen' ),
```

Comment:
I doubled checked with @melchoyce; the reason is to make sure it matches other title case strings in the theme, like 'Leave a Reply'.